### ISSUE TITLE
500 Error when logging into ESPM from SEED

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1654]
### Expected Behavior
Log into ESPM account from SEED
### Actual Behavior 
Got an error when trying to do this
### Steps to Reproduce
See this doc
https://drive.google.com/open?id=12FvCxHNs8ZXpQuP_2vs8vMn8Xg7mmTOn
### Instance Information
Server: dev1
SHA: 90e4a7d

**Comment by RDmitchell** [comment]
@Myoldmopar -- SF is having the same problem they originally did logging into ESPM from SEED. The details are in the doc referenced above. Thx

**Comment by RDmitchell** [comment]
Server: dev1
SHA: 90e4a7d

I just tested logging into our (simple) ESPM test account and it worked.

**Comment by RDmitchell** [comment]
We will probably have to work directly with SF to figure out the subtleties of what is going on.

**Comment by RDmitchell** [comment]
Both SF and Berkeley are having this problem.

**Comment by nllong** [comment]
test on dev1

**Comment by RDmitchell** [comment]
instance: dev1
SHA: 6639ad8
Org: LBNL 20

From the LBNL ESPM test account that I set up, I imported data from the "Data Request:SEED City Test Report April 24 - Test 2" under "SEED City Test Report".
![image](https://user-images.githubusercontent.com/6314950/45389430-c3f15c00-b5d0-11e8-8bd5-8032558aeb66.png)
The 4 records imported properly into SEED
So I am not getting this error anymore.

However, when I compare, field by field, the data imported into SEED vs the data in the file downloaded from ESPM, two fields are missing from the SEED import -- 
* Site EUI (kBtu/ft²)
* Source EUI (kBtu/ft²)

I will make a separate issue for that, since this issue is about a 500 error which doesn't seem to be happening anymore. 

The real test for this issue, however, is if San Francisco can get their large data set imported directly from ESPM, which it seems they can.
